src/main/resources/application-dev.properties

```diff
@@ -1,5 +1,7 @@
 spring.datasource.url=jdbc:postgresql://localhost:5050/et_cos
 
+spring.flyway.locations=classpath:db/migration,classpath:db/dev
```

Comment:
isn't this also needed in application.properties as well?

Reply:
Migration scripts are read from db/migration by default.

https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto.data-initialization.migration-tool.flyway